### ISSUE TITLE
All languages given in JSON-LD language map, even if no values

### ISSUE THREAD

**Issue by azaroth42** [archesproject/arches#9192]
**Arches Version/Branch:** stable/7.2.1

**Describe the bug and how to reproduce it:**

When requesting a resource that has a string data type in the model, then the property is always present and the language map has all of the known languages with an empty string `@value`.

For example:

```
{"@context": "https://linked.art/ns/v1/linked-art.json", "id": "http://127.0.0.1:8000/resources/4ca0beb8-e282-4b3e-a0ee-bec522b59fbe", "identified_by": [{"_label": [{"@language": "fr", "@value": ""}, {"@language": "de", "@value": ""}, {"@language": "el", "@value": ""}, {"@language": "ru", "@value": ""}, {"@language": "en-us", "@value": ""}, {"@language": "en", "@value": ""}, {"@language": "zh", "@value": ""}, {"@language": "pt", "@value": ""}], "content": [{"@language": "de", "@value": ""}, {"@language": "en", "@value": "test"}, {"@language": "el", "@value": ""}, {"@language": "ru", "@value": ""}, {"@language": "zh", "@value": ""}, {"@language": "en-us", "@value": ""}, {"@language": "pt", "@value": ""}, {"@language": "fr", "@value": ""}], "id": "http://127.0.0.1:8000/tile/849b9deb-4ca5-49e1-a01d-09a507c13af9/node/bb341246-9358-11ed-a690-acde48001122", "type": "Name"}], "type": "Type"}
```

Should actually be just

```
{"@context": "https://linked.art/ns/v1/linked-art.json", "id": "http://127.0.0.1:8000/resources/4ca0beb8-e282-4b3e-a0ee-bec522b59fbe", "identified_by": [ {"type": "Name", "content": [ {"@language": "en", "@value": "test"}]} ], "type": "Type"}
```

To Reproduce:

* Create a model with a string datatype
* Request the JSON-LD representation

**Comment by azaroth42** [comment]
The fix:

In arches/app/datatypes/datatypes.py, line 214, add a test in the for loop:

```
            for key in edge_info["range_tile_data"].keys():
                if edge_info["range_tile_data"][key]["value"]:
                    g.add((edge_info["d_uri"], URIRef(edge.ontologyproperty), Literal(edge_info["range_tile_data"][key]["value"], lang=key)))
```

(I made a PR, but then closed as wasn't sure which branch to send it to. Look at the branch 9192_empty_languages for the change in situ)

**Comment by apeters** [comment]
closed by #10033
